README.md

```diff
@@ -7,7 +7,16 @@
 
 A small Python library supporting content-negotiation.
 
-It is used to decide content type based on a list of media ranges, as well as deciding content-language based on the accpe-language header.
+It is used to decide content type based on a list of media ranges in the accept header, as well as deciding content-language based on the accept-language header.
+
+* Media ranges/language ranges with a q-value of 0.0 will be ignored.
+* Q-values above 1.0 will be treated as 1.0. Q-values below 0.0 will be treated as 0.0.
+* When a media range is not specified, it will be treated as */*.
+* When a language range is not specified, it will be treated as *.
```

Comment:
`*/*` i markdown ser slik ut: */*.
```suggestion
* When a media range is not specified, it will be treated as `*/*`.
* When a language range is not specified, it will be treated as `*`.
```